hyperstart.go

```diff
@@ -193,17 +178,18 @@ func (h *hyper) bindUnmountAllRootfs() {
 }
 
 // init is the agent initialization implementation for hyperstart.
-func (h *hyper) init(pod Pod, config interface{}) error {
+func (h *hyper) init(pod *Pod, config interface{}) error {
```

Comment:
Why do we need a pod pointer ? For this:

`pod.config.AgentConfig = h.config` ?